packages/core/admin/admin/src/components/NpsSurvey/index.js

```diff
@@ -124,6 +125,8 @@ const NpsSurvey = () => {
   const { formatMessage } = useIntl();
   const { npsSurveySettings, setNpsSurveySettings } = useNpsSurveySettings();
   const [isFeedbackResponse, setIsFeedbackResponse] = React.useState(false);
+  const [isSendingSurvey, setIsSendingSurvey] = React.useState(false);
```

Comment:
Do you think it would make sense to use a react-query mutation instead? This will give us everything we need in terms of error and loading state handling already. Since we are slowly migrating the codebase to it I'd prefer that (apologies I didn't think of this with my last comment).